Dockerfile

```diff
@@ -4,20 +4,14 @@ ENV PNPM_HOME="/pnpm"
 ENV PATH="$PNPM_HOME:$PATH"
 RUN corepack enable
 
-FROM base AS prod
-
 WORKDIR /app
 COPY . .
 
 RUN pnpm install --frozen-lockfile
-RUN pnpm run build
-
-FROM nginx:stable-alpine
+RUN apk add --no-cache nginx
 
 COPY nginx.conf /etc/nginx/conf.d/default.conf
 
-COPY --from=prod /app/dist/ /app
-
 EXPOSE 80
 
-CMD [ "nginx" ]
+CMD [ "cd /app && pnpm run build && nginx -g 'daemon off;'" ]
```

Comment:
In that case, you will wait the container startup till your static files will be compiled. I'm not sure that we should do it.

Reply:
env can only be obtained when the container starts, so I have to.